src/test/java/ladder/domain/LadderTest.java

```diff
@@ -0,0 +1,23 @@
+package ladder.domain;
+
+import ladder.domain.RowGeneratorTest.SequenceRowGenerator;
+import org.assertj.core.api.SoftAssertions;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+public class LadderTest {
+    @ParameterizedTest
+    @ValueSource(ints = {1, 2, 3, 4, 5, 6})
+    public void 사다리는_높이_수만큼_행을_가진다(int height) {
+        // Given
+        Ladder ladder = new Ladder(1, new Height(height), new SequenceRowGenerator());
+
+        // When
+
+        // Then
+        SoftAssertions.assertSoftly(softly -> {
+            softly.assertThat(ladder).isNotNull();
+            softly.assertThat(ladder.getRowSize()).isEqualTo(height);
+        });
```

Comment:
```suggestion
        // Given
        SequenceRowGenerator rowGenerator = new SequenceRowGenerator();
        Height height = new Height(value);
        
        // When
        Ladder ladder = new Ladder(1, height, rowGenerator);

        // Then
        assertSoftly(softly -> {
            softly.assertThat(ladder).isNotNull();
            softly.assertThat(ladder.getRowSize()).isEqualTo(height);
        });
```
when에서 사다리를 생성해야 할 것 같아요
given-when-then은 사전작업-검증 행위-검증 인 것을 기억하세요!

SoftAssertions 사용 👍 (static import를 하면 더 좋습니다)